mise.toml

```diff
@@ -0,0 +1,3 @@
+[tools]
+go = "1.23.0"
+golangci-lint = "1.64.8"
```

Comment:
This feels a bit too opinionated for an open source project. I'd rather avoid having to keep versions in sync in multiple locations. If it can't be updated automatically, probably best in the .gitignore file.